views/createPoll.mustache

```diff
@@ -0,0 +1,20 @@
+<html>
+    <head>
+        <title>Register</title>
+    </head>
+    <body>
+    {{title}}
+       <form id="createPoll_form">
+        <legend>Poll</legend>
+        <input type="submit"><br>
+        <input type="text" id="question" name="question" placeholder="question" required><br>
+        <input type="text" id="choice1" name="choice1" placeholder="choice1" required><br>
+        <input type="text" id="choice2" name="choice2" placeholder="choice2" required><br>
+        <input type="text" id="choice3" name="choice3" placeholder="choice3" required><br>
+        <input type="text" id="choice4" name="choice4" placeholder="choice4" required><br>
+        <input type="text" id="choice5" name="choice5" placeholder="choice5" required><br>
+      </form>
+        <button id="addChoice">Add Choice</a>
```

Comment:
This button doesn't seem to currently do anything. Is the intent that it would add an additional choice form input every time the button is pressed?